# Description: en_CA date format should be "%y-%m-%d" (see http://inms-ienm.nrc-cnrc.gc.ca/faq_time_e.html#Q8)
# Ubuntu: https://launchpad.net/bugs/214730
# Upstream: http://sourceware.org/bugzilla/show_bug.cgi?id=9842
--- langpack-locales-2.9+cvs20090214/locales/en_CA	2009-02-13 19:19:33.000000000 +0100
+++ langpack-locales-2.9+cvs20090214.new/locales/en_CA	2009-02-24 17:29:04.557636635 +0100
@@ -125,7 +125,7 @@
             "<U004E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
             "<U0044><U0065><U0063><U0065><U006D><U0062><U0065><U0072>"
 d_t_fmt     "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0072><U0020><U0025><U005A>"
-d_fmt       "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
+d_fmt       "<U0025><U0079><U002D><U0025><U006D><U002D><U0025><U0064>"
 t_fmt       "<U0025><U0072>"
 am_pm       "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U0070>"
